tcms/core/tests/test_views.py

```diff
@@ -8,43 +8,6 @@
 
 from tcms import urls
 from tcms.tests import BaseCaseRun
-from tcms.tests.factories import (
-    TestExecutionFactory,
-    TestPlanFactory,
```

Comment:
Some of these imports are still in use, see the failing checks at https://github.com/kiwitcms/Kiwi/pull/2239/checks?check_run_id=1839431769 and https://github.com/kiwitcms/Kiwi/pull/2239/checks?check_run_id=1839432801 for example.